src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style.css';
+import List, { editTask, cleanAll } from './crud.js';
+import state from './interactive.js';
+
```

Comment:
:face_with_spiral_eyes:  This is optional comments

It is always good to include comments in your code to help other developers understand your code better. The code could benefit from comments that explain what each section of the code does.

eg: here you should put :  // Define DOM elements